app/models/concerns/person_with_dob.rb

```diff
@@ -1,9 +0,0 @@
-module PersonWithDob
```

Comment:
This method does not consider the date_stamp it's use was the source of the regression.  There term under18 has been deprecated in favour of the already existing `age_passported?`